scripts/populate.js

```diff
@@ -3,7 +3,17 @@ var Chatty = (function(originalChatty){
 	var outputString = "";
 	var messagesArray = [];
 
-	
+	Chatty.getMessage = function () {
+		return messagesArray;
+	}
+
+	Chatty.newMessage = function () {
+		messagesArray.push(inputEl.value);
+		for (var i = 0; i < messagesArray.length; i++) {
+			outputString = messagesArray[i];
```

Comment:
This is the part that generated your blank strings. When you pushed strings into the array you were pushing the value of inputEl, which by default was blank. The way the xhr.js file formatted it passes the five strings and the element we want them in to getMessage one at a time anyway, so we don't even need to loop over the array every time we want to add a message, just add the new message string to it.